src/openstudio_lib/OSItemList.hpp

```diff
@@ -105,7 +103,6 @@ class OSItemList : public OSItemSelector
   bool m_itemsDraggable;
   bool m_itemsRemoveable;
   OSItemType m_type;
-  bool m_dirty;
```

Comment:
`m_dirty` wasn't actually controlling anything here, all that `refresh` was doing was to clear `m_dirty`